src/components/sections/social-media-links.svelte

```diff
@@ -0,0 +1,137 @@
+        d="M51.5213 0H4.785C2.23125 0 0 1.8375 0 4.36125V51.2012C0 53.7387 2.23125 56 4.785 56H51.5075C54.075 56 56 53.7237 56 51.2012V4.36125C56.015 1.8375 54.075 0 51.5213 0ZM17.3587 46.6787H9.33625V21.735H17.3587V46.6787ZM13.625 17.9425H13.5675C11 17.9425 9.3375 16.0312 9.3375 13.6388C9.3375 11.2025 11.0438 9.33625 13.6688 9.33625C16.2938 9.33625 17.9 11.1888 17.9575 13.6388C17.9563 16.0312 16.2938 17.9425 13.625 17.9425ZM46.6787 46.6787H38.6562V33.04C38.6562 29.7725 37.4888 27.54 34.5863 27.54C32.3688 27.54 31.0562 29.04 30.4725 30.5013C30.2537 31.0263 30.195 31.7412 30.195 32.4712V46.6787H22.1725V21.735H30.195V25.2062C31.3625 23.5437 33.1862 21.1513 37.43 21.1513C42.6963 21.1513 46.68 24.6225 46.68 32.1063L46.6787 46.6787Z"
+        fill="white"></path>
+    </svg>
+
+    linkedin</a
+  >
+  <a
+    href=" https://www.youtube.com/channel/UCDMBj0o4V8Cpt0jgsZLNVVg"
+    class="social-link"
+    target="”_blank”"
+  >
+    <svg
+      width="64"
+      height="49"
+      viewBox="0 0 64 49"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M63.5798 10.5987C63.5798 4.97375 59.4423 0.44875 54.3298 0.44875C47.4048 0.125 40.3423 0 33.1248 0H30.8748C23.6748 0 16.5998 0.125 9.67478 0.45C4.57478 0.45 0.437284 5 0.437284 10.625C0.124784 15.0737 -0.00771643 19.5238 -0.000216427 23.9737C-0.0127164 28.4237 0.12895 32.8779 0.424784 37.3363C0.424784 42.9613 4.56228 47.5238 9.66228 47.5238C16.9373 47.8613 24.3998 48.0112 31.9873 47.9987C39.5873 48.0238 47.0289 47.8654 54.3123 47.5238C59.4248 47.5238 63.5623 42.9613 63.5623 37.3363C63.8623 32.8737 63.9998 28.4237 63.9873 23.9613C64.0156 19.5112 63.8798 15.0571 63.5798 10.5987ZM25.8748 36.2363V11.6737L43.9998 23.9487L25.8748 36.2363Z"
+        fill="white"></path>
+    </svg>
+    youtube</a
+  >
+</section>
+
+<style>
+  .social-media-links-section {
+    display: flex;
+    flex-direction: column;
+    margin: 0 32px;
+    align-items: center;
+  }
+  .social-link {
+    cursor: pointer;
+    border-radius: 1em;
+    background-color: var(--acm-dark);
+    padding: 8px 16px 8px 16px;
+    margin: 1em 2em;
+    color: var(--acm-light);
+    text-decoration: none;
+    border: 2px solid var(--acm-dark);
+    font-size: 1em;
+    text-transform: lowercase;
+    text-align: right;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 75%;
+  }
+  .social-link svg {
+    width: 1.5em;
+    height: 1.5em;
+  }
+
+  .social-link:hover {
+    transition: color 0.25s ease-in-out, background-color 0.25s ease-in-out;
+    color: var(--acm-blue);
+    background-color: var(--acm-light);
+  }
+
+  .social-link:hover svg path {
+    transition: fill 0.25s ease-in-out;
+    fill: var(--acm-blue);
+  }
+
+  @media screen and (min-width: 768px) {
+    .social-media-links-section {
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: space-around;
+    }
+    .social-link {
+      width: 40%;
+      font-size: 1.5em;
```

Comment:
This line can be omitted since the `--body-font-size` variable automatically resizes itself accordingly.